bili.config.ts

```diff
@@ -0,0 +1,21 @@
+import { Config } from 'bili'
+
+const config: Config = {
+  plugins: {
+    typescript2: {
+      // Override the config in `tsconfig.json`
+      tsconfigOverride: {
+        include: ['src']
+      }
+    }
+  },
+
+  // Let's take this opportunity to move the CLI flags here as well
+  input: 'src/index.ts',
+  output: {
+    dir: '.',
+    format: ['cjs', 'esm']
```

Comment:
```suggestion
    format: ['cjs', 'umd', 'esm'],
    moduleName: 'cursornext',
    sourceMap: true,
    sourceMapExcludeSources: true
```
I would like to add `umd` output format and sourcemap also.